src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -1747,7 +1747,7 @@ test_compatible_null_error_pointer (void)
    mock_server_run (server);
    client =
       test_framework_client_new_from_uri (mock_server_get_uri (server), NULL);
-   td = mc_tpld_unsafe_get_const (client->topology);
+   mc_tpld_unsafe_get_const (client->topology);
```

Comment:
This line can be deleted outright.

Reply:
@vector-of-bool Thanks for pointing that out.  I assumed that the call to
`mc_tpld_unsafe_get_const` produced a side-effect that might not be okay
to eliminate.  I just looked at the implementation and I can see that it
does not have any side-effects.